service/s3/s3manager/batch.go

```diff
@@ -382,4 +396,6 @@ func (batcher *UploadObjectsIterator) UploadObject() BatchUploadObject {
 // BatchUploadObject contains all necessary information to run a batch operation once.
 type BatchUploadObject struct {
 	Object *UploadInput
+	// After will run after each iteration during the batch process
```

Comment:
I'll clarify this statement to stay after the object has been attempted to be uploaded. Regardless of success/failure.

Same for the other `After` functions